vllm/v1/core/sched/scheduler.py

```diff
@@ -464,9 +484,12 @@ def schedule(self) -> SchedulerOutput:
                 # This information is used to determine if a load is
                 # needed for this request.
                 if self.connector is not None:
+                    update_blocks = new_computed_blocks + new_blocks
```

Comment:
I think it is better for us to disable KVConnector + Encoder/Decoder for this PR. We have not really tested that it works so I would rather we fail for now